### DISCUSSION TITLE
Moving forward to modular playbooks and roles with galaxy collections?

### DISCUSSION THREAD

**Discussion by stoeps13** [HCL-TECH-SOFTWARE/connections-automation#166] [Ideas]
Hi
I'm using Ansible for most of my projects since several years, so I enjoy this repo a lot, but it is hard to include the roles and playbooks into existing projects or add local roles to get specialities of an environment in the code with breaking the option to update (pull) this repository frequently or just building a pull request because of the differences.
Have you ever thought about moving this repository in Ansible Galaxy Roles and Collections, so we can add them through requirements.yml to any Ansible project? Way easier to maintain and test than having this as a mono repository.

https://docs.ansible.com/ansible/latest/user_guide/collections_using.html

Regards
Christoph

**Comment by gyanenduprakash** [comment]
Thanks for suggestion , we will have discussion with team !!
